MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -92,4 +92,18 @@ open class BaseInstructionsBannerView: UIControl {
         
         distance = 100
     }
+    
+    /**
+     Updates the instructions banner for a given `RouteProgress`.
+     */
+    public func update(for routeProgress: RouteProgress) {
+        let stepProgress = routeProgress.currentLegProgress.currentStepProgress
+        let distanceRemaining = stepProgress.distanceRemaining
+        
+        guard let visualInstruction = routeProgress.currentLegProgress.currentStep.instructionsDisplayedAlongStep?.last else { return }
+        
+        set(visualInstruction.primaryTextComponents, secondaryInstruction: visualInstruction.secondaryTextComponents)
```

Comment:
Argh, this `set(_:secondaryInstruction:)` method again. Can we break it out into two properties that can be set independently? If we really want to prevent one from being set without the other, then we should replace `set(_:secondaryInstruction:)` with a `update(for:)` method that takes a single VisualInstruction.